mne_connectivity/viz/_3d.py

```diff
@@ -37,9 +44,13 @@ def plot_sensors_connectivity(
     n_con : int
         Number of strongest connections shown. By default 20.
     cmap : str | instance of matplotlib.colors.Colormap
-        Colormap for coloring connections by strength. If :class:`str`, must
-        be a valid Matplotlib colormap (i.e. a valid key of
-        ``matplotlib.colormaps``). Default is ``"RdBu"``.
+        Colormap for coloring connections by strength. If :class:`str`, must be a valid
+        Matplotlib colormap (i.e. a valid key of `matplotlib.colormaps`). Default is
+        ``"RdBu"``.
+    min_distance : float
+        The minimum distance required between two sensors to plot a connection between
+        them, in meters. Default is 0.05 (i.e. 5 cm).
+        .. versionadded:: 0.8
```

Comment:
Pretty sure for this to render properly it needs a newline in between
```suggestion

        .. versionadded:: 0.8
```

Reply:
You're right, sorry, was too hasty